less/CardMatchHeight.less

```diff
@@ -0,0 +1,14 @@
+.card-pf-match-height {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  .card-pf-heading {
+    margin: 0 -20px;
+    .card-pf-time-frame-filter {
+      margin-top: 15px;
+    }
+  }
+}
+.card-pf-body {
+  flex: 1;
+}
```

Comment:
I suggest replacing all of this with the following instead:
```
.card-pf-match-height[style*="height"] {
  position: relative;

  .card-pf-footer {
    position: absolute;
    bottom: 0;
    width: 100%;
  }
}
```
AND you can remove the inline `position: relative;` from the `card-pf` element since it's in the css. 

The issue with setting `display: flex;` on the card is that [margins do not collapse for flex items](https://www.w3.org/TR/css-flexbox-1/#flex-containers) and the current css/html of cards are dependent on collapsing margins. The simplest work-around I could come up with is to absolute position the footer to the bottom, once the height has been defined for the card.

Also, on slack you had asked about being able to set a custom height, but I haven't seen that anywhere in patternfly, and in playing around with setting the height, it doesn't look like the current css accommodates that. For example, if the height of the card is less than the height of the body, then the body contents will overflow the card.

Reply:
Thank you for the detailed answer! ill work with all of this on Sunday! until then, do we even want to allow a consumer to create a card with custom height?

Reply:
Since the pattern doesn't support that use case today, then I don't think we should include that as an example. I don't think we need to prevent consumers from setting the height either if that's something they need. I just expect that it's on the consumer to fully test and handle any css issues that come with doing that.